### PR TITLE
Set up hosting

### PR DESCRIPTION
**Pull request by wenche**
Resolves AB#84239

### CODE REVIEW

File: .github/workflows/build-and-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Build and deploy
+on:
+  pull_request:
+    branches:
+      - main
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  AZURE_RESOURCE_GROUP_NAME: ""
+  AZURE_FUNCTIONAPP_NAME: "datamarket-place-adapter-service"
```
Comment (aronhoyer) [inline .github/workflows/build-and-deploy.yaml:9]: data-marketplace 👀

Comment (wenche) [reply .github/workflows/build-and-deploy.yaml]: Bare sjekker om du er våken :D

**Review by nilsml — APPROVED** [review]
Good stuff!

File: .gitignore [hunk 2]
```diff
@@ -46,3 +46,10 @@ __pycache__/
 __blobstorage__
 __queuestorage__
 __azurite_db*__.json
+
+### VisualStudioCode ###
+.vscode/**/*
```
Comment (aronhoyer) [inline .gitignore:51]: why add this, and exclude all the files you've added?

**Review by aronhoyer — APPROVED** [review]
lgtm 👍 why add all the vscode configs though? 🤔

**Pull request merged**